domains/certificates/Query.API/API.IntegrationTests/CertificateIssuingTests.cs

```diff
@@ -68,16 +71,18 @@ public async Task GetList_NoCertificates_ReturnsNoContent()
         response.StatusCode.Should().Be(HttpStatusCode.NoContent);
     }
 
+    //FiveMeasurements tests cannot be run first, then they fail, so this is set to run first.
```

Comment:
Why?

Reply:
Could not figure out why. It happens on main as well

Reply:
It is a bug in the Wallet. Here are logs showing 4 errors due to `duplicate key value violates unique constraint \"registries_name_key\"`:

![image](https://github.com/Energinet-DataHub/energy-origin/assets/12843334/1b85d139-b8b0-4de5-bf74-a23eb15ddfdc)

Reply:
I don't see why that hits these tests. How do you suggest we proceed?

Reply:
It is a bug in the wallet version 0.2.1 when creating multiple certificates for the same registry and the wallet has no registries defined.
That is fixed in wallet version 0.3.0

Reply:
I am closing this and when the wallet is updated to 0.3.0, we can remove the work-around